gamestonk_terminal/portfolio/portfolio_view.py

```diff
@@ -222,17 +230,16 @@ def plot_ef(
     ax.scatter(stds, rets, marker=".", c=sharpes, cmap="viridis_r")
     plotting.plot_efficient_frontier(ef, ax=ax, show_assets=True)
     # Find the tangency portfolio
-    rfrate = get_rf()
-    ret_sharpe, std_sharpe, _ = ef.portfolio_performance(risk_free_rate=rfrate)
+    ret_sharpe, std_sharpe, _ = ef.portfolio_performance(risk_free_rate=rf)
```

Comment:
I'd stick to rfrate as a name.

Descriptive names very much improve readability.